examples/app.py

```diff
@@ -31,10 +31,11 @@
 
     .. code-block:: console
 
-        cdvirtualenv src/invenio-marc21
         pip install -e .[all]
-        pip install invenio-theme==1.0.0a9
+        pip install git+git://github.com/inveniosoftware/dojson.git
```

Comment:
It should but after the new version of dojson is released (the current released version is missing some cli functions) :/